### ISSUE TITLE
Multiple re-rendering on same page

### ISSUE THREAD

**Issue by emorling** [k-128/react-tradingview-embed#18]
It seems that one of the widgets get re-rendering due to props being changed, all the TradingView are reloaded as well. Is this to be expected?

**Comment by diazidabagus** [comment]
Yes I'm having the same issue, this triggers unnecessary reloading
